Formula/slither-analyzer.rb

```diff
@@ -0,0 +1,55 @@
+class SlitherAnalyzer < Formula
+  include Language::Python::Virtualenv
+
+  desc "Solidity static analysis framework written in Python 3"
+  homepage "https://github.com/crytic/slither"
+  url "https://github.com/crytic/slither/archive/refs/tags/0.8.3.tar.gz"
+  sha256 "7549ab983541094a34525d7e74af1fa9b1538122c92a6ed969d2e4c7fcc7b42b"
+  license "AGPL-3.0-only"
+  head "https://github.com/crytic/slither.git", branch: "master"
+
+  depends_on "crytic-compile"
+  depends_on "python@3.10"
+  depends_on "solc-select"
+
+  resource "prettytable" do
+    url "https://files.pythonhosted.org/packages/10/88/ef38a6e4bc375600d3031e405a8d3b3dc4a154fccffd21d5d06e66c96230/prettytable-3.3.0.tar.gz"
+    sha256 "118eb54fd2794049b810893653b20952349df6d3bc1764e7facd8a18064fa9b0"
+  end
+
+  resource "pysha3" do
+    url "https://files.pythonhosted.org/packages/73/bf/978d424ac6c9076d73b8fdc8ab8ad46f98af0c34669d736b1d83c758afee/pysha3-1.0.2.tar.gz"
+    sha256 "fe988e73f2ce6d947220624f04d467faf05f1bbdbc64b0a201296bb3af92739e"
+  end
```

Comment:
I think this might not be needed anymore, since it's in `crytic-compile`'s virtualenv.

Reply:
Makes sense, yep!